build/cloudbuild_prow.yaml

```diff
@@ -0,0 +1,18 @@
+#  Copyright 2018 The Kubernetes Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+steps:
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/kubebuilder/prow-base', '.']
+images: ['gcr.io/kubebuilder/prow-base']
```

Comment:
oh, heh.  If this is all we're doing, we can tell cloudbuild to do this directly.

Reply:
this is fine, though

Reply:
tweak the hardcoded `kubebuilder` to be `$PROJECT_ID` and this should be set.

Reply:
(should've mentioned that earlier -- my fault)

Reply:
I've pushed that version for you to test with -- we'll set up an autobuild later.

Reply:
Guess we didn't end up needing this 🙂 We can set up a nice image with our deps + auto-bumping if we want, but seems like we're okay for now?